### ISSUE TITLE
RangeError: Maximum call stack size exceeded with version 1.8.5

### ISSUE THREAD

**Issue by binary64** [shuckster/eslint-plugin-big-number-rules#4]
```
Oops! Something went wrong! :(

ESLint: 8.15.0

RangeError: Maximum call stack size exceeded
Occurred while linting C:\p\monorepo\apps\web\assets\notificationData.tsx:30
Rule: "big-number-rules/parseFloat"
    at Function.[Symbol.hasInstance] (<anonymous>)
    at C:\p\monorepo\node_modules\eslint-plugin-big-number-rules\dist\index.js:7:1208

```

**Comment by shuckster** [comment]
Thanks for the report. Can you share any code or steps to reproduce?

**Comment by shuckster** [comment]
@binary64 - Please try `1.9.0`.

I've removed the build process to hopefully improve the error-messages you are getting.

**Comment by binary64** [comment]
I have tried 1.9.0 but still see the same:
```
Oops! Something went wrong! :(

ESLint: 8.16.0

RangeError: Maximum call stack size exceeded
Occurred while linting C:\p\monorepo\apps\web\assets\notificationData.tsx:30
Rule: "big-number-rules/parseFloat"
    at C:\p\monorepo\node_modules\match-iz\dist\index.js:7:2450
    at Array.every (<anonymous>)
    at r (C:\p\monorepo\node_modules\match-iz\dist\index.js:7:2437)
    at C:\p\monorepo\node_modules\match-iz\dist\index.js:7:2372
    at Array.every (<anonymous>)
    at r (C:\p\monorepo\node_modules\match-iz\dist\index.js:7:2363)
    at C:\p\monorepo\node_modules\match-iz\dist\index.js:7:2372
    at Array.every (<anonymous>)
    at r (C:\p\monorepo\node_modules\match-iz\dist\index.js:7:2363)
    at C:\p\monorepo\node_modules\match-iz\dist\index.js:7:2372
```

**Comment by binary64** [comment]
Minimum reproduction:
```
const notificationData = ({ t }) => {
  return t()
}

export default notificationData
```

**Comment by shuckster** [comment]
Thank you, I think I've found the source of the problem.

Please try `1.9.1`.

**Comment by binary64** [comment]
Thanks, this issue appears fixed in the latest version 1.9.1

**Comment by shuckster** [comment]
Thanks again!
